lua/fidget/progress/handle.lua

```diff
@@ -0,0 +1,145 @@
+local notification = require("fidget.notification")
+
+local NEXT_ID = 0
+local PREFIX = "fidget-"
```

Comment:
Nit: the token prefix could be more specific, e.g., `fidget-progress-handle-`. Also these variables shouldn't be capitalized imo